app/components/status_tags/reviewer_assessment_detail_component.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module StatusTags
+  class ReviewerAssessmentDetailComponent < StatusTags::BaseComponent
+    def initialize(assessment_detail:)
+      @assessment_detail = assessment_detail
+    end
+
+    private
+
+    attr_reader :assessment_detail
+
+    def status
+      :updated if assessment_detail.updated?
```

Comment:
There's another ticket which the design team are working on to clarify the other statuses that should show up here:

https://trello.com/c/DdrC2wdW/1344-review-assessment-summary-page-status-tags